tests/dft-custom-dhdf/input.dat

```diff
@@ -87,6 +78,23 @@ compare_values(-0.062529936, e_mp2, 5,  'Ne: PWPB95_PT2')   #TEST
 compare_values(-128.920706595202, edhdf, 5,  'Ne: PWPB95')   #TEST
 clean()
 
+set freeze_core true 
+edhdf = energy('dsd-blyp')
+e_dft=get_variable('SCF TOTAL ENERGY')
+e_mp2=get_variable('DOUBLE-HYBRID CORRECTION ENERGY')
+compare_values(-128.75560404, e_dft, 6, 'Ne: DSD-BLYP_DFT')   #TEST
+compare_values(-0.126886573, e_mp2, 7,  'Ne: DSD-BLYP_PT2')   #TEST 
+compare_values(-128.882490613961, edhdf, 6,  'Ne: DSD-BLYP')   #TEST
+clean()
+
+edhdf = energy('pbe0-dh')
```

Comment:
In definition above says `pbe0-dh` is disabled, and it is disabled in the superfuncs dict. Yet testing it here. Is there an inconsistency or how does this work?